app/templates/components/questionnaire/job-name.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="row">
+    <div class="col-md-6">
```

Comment:
I removed these extra divs. Bootstrap standard text input always seems excessively wide, but I guess that's its style.